rudderstack/analytics/request.py

```diff
@@ -1,29 +1,44 @@
 from datetime import date, datetime
-from dateutil.tz import tzutc
+from io import BytesIO
+from gzip import GzipFile
 import logging
 import json
+from dateutil.tz import tzutc
 from requests.auth import HTTPBasicAuth
 from requests import sessions
 
-from rudder_analytics.version import VERSION
-from rudder_analytics.utils import remove_trailing_slash
+from rudderstack.analytics.version import VERSION
+from rudderstack.analytics.utils import remove_trailing_slash
 
 _session = sessions.Session()
 
 
-def post(write_key, host=None, timeout=15, **kwargs):
+def post(write_key, host=None, gzip=True, timeout=15, proxies=None, **kwargs):
     """Post the `kwargs` to the API"""
-    log = logging.getLogger('rudder')
+    log = logging.getLogger('rudderstack')
     body = kwargs
     body["sentAt"] = datetime.utcnow().replace(tzinfo=tzutc()).isoformat()
-    url = remove_trailing_slash(host or 'https://hosted.rudderlabs.com') + '/v1/batch'
+    url = remove_trailing_slash(host or 'https://api.rudderstack.io') + '/v1/batch'
```

Comment:
this url is wrong rudderstack.io not existing

Reply:
changed to api.rudderstack.com